typebeast.yml

```diff
@@ -152,7 +152,7 @@ wysiwyg: # optional, 1.0
     h4: h4
     h5: h5
     h6: h6
-    lede: '.with-lede p:first-child'
+    lede: '&.with-lede p:first-child' # should have leading &
```

Comment:
inspect selector and, if leading ampersand, split it off and omit leading space in scoped selector